Cargo.toml

```diff
@@ -33,16 +33,16 @@ codegen-units = 1
 lto = true
 
 [workspace.dependencies]
-assembly = { package = "miden-assembly", version = "0.11", default-features = false }
+assembly = { package = "miden-assembly", git = "https://github.com/0xPolygonMiden/miden-vm.git", branch = "next", default-features = false }
```

Comment:
Just to note: We should either pin the dependencies to a specific commit on next to avoid breakage if next is updated, or merge #1059 shortly after this one (which points to the `element-addressable-memory` branch).

Reply:
I hope that we will be able to merge the element-addressable memory PR right after this one, but I agree that it makes sense to pin the specific commit